config/config.example.yml

```diff
@@ -169,6 +169,10 @@ browseBy:
   # The absolute lowest year to display in the dropdown (only used when no lowest date can be found for all items)
   defaultLowerLimit: 1900
 
+browseCommunities:
+  communityListPageSize: 20
+  topLevelPageSize: 5
```

Comment:
We discussed it in the DSpace 7 Workgroup meeting of Sept 1st, and in order to keep this kind of config consistent across pages, please structure it something like:

```yaml
communityListPage:
  pageSize: 20
homePage:
  topLevelCommunityList:
    pageSize: 5
```